data/json/items/armor/integrated.json

```diff
@@ -938,20 +937,10 @@
     "material": [ "plastic", "lvl4ballisticglass" ],
     "symbol": "[",
     "color": "dark_gray",
-    "warmth": 5,
+    "warmth": 0,
     "material_thickness": 1.5,
     "environmental_protection": 10,
-    "flags": [
-      "INTEGRATED",
-      "UNBREAKABLE",
-      "SKINTIGHT",
-      "SOFT",
-      "WATER_FRIENDLY",
-      "SUN_GLASSES",
-      "NO_REPAIR",
-      "SWIM_GOGGLES",
-      "NO_SALVAGE"
-    ],
+    "flags": [ "INTEGRATED", "UNBREAKABLE", "PERSONAL", "SOFT", "WATER_FRIENDLY", "SUN_GLASSES", "NO_REPAIR", "SWIM_GOGGLES" ],
```

Comment:
Why remove NO_SALVAGE?

Reply:
I don't remember the original reason. I can put it back but integrated armors will automatically have NO_SALVAGE nowadays.

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/pull/64449 suggests that is not the case?

Reply:
You are right. I assumed that pr fixed it that way but it didn't.